qa/tasks/ceph_manager.py

```diff
@@ -1371,7 +1371,9 @@ def raw_cluster_cmd_result(self, *args, **kwargs):
         """
         Start ceph on a cluster.  Return success or failure information.
         """
-        kwargs['args'], kwargs['check_status'] = args, False
+        if kwargs.get('args') is None and args:
+           kwargs['args'] = args
```

Comment:
The code seems contrary to the title in your commit comment ?

Reply:
I don't see a contradiction... The current code at master does `kwargs['args'] = args` unconditionally. This commit modifies it to only do it conditionally. Please let me know if I should reword my commit message.